tests/karma/unit/filters/relative-date.js

```diff
@@ -42,7 +42,7 @@ describe('relativeDate filter', function() {
     chai.expect(element.text()).to.equal('somerelativetime');
   });
 
-  it('should render a time when the date is today', () => {
+  it('should render a time when the date is today', function() {
```

Comment:
This is a step backwards. Ideally this PR would not contain changes in any files which are purely whitespace or formatting to try and keep the git history relevant. I suspect some of this is done by your automatic formatter.